crates/kitsune_p2p/kitsune_p2p/Cargo.toml

```diff
@@ -43,7 +43,7 @@ thiserror = "1.0.22"
 tokio = { version = "1.27", features = ["full"] }
 tracing = "0.1"
 tokio-stream = "0.1"
-tx5 = { version = "=0.0.1-alpha.18", optional = true }
+tx5 = { version = "=0.0.2-alpha", optional = true }
```

Comment:
per https://github.com/holochain/holochain/pull/2669#discussion_r1296006148 please update the other tx5 refs as well.